devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -53,8 +56,20 @@ metadata {
 }
 
 def initialize() {
-	// Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
-	sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	if (zwaveInfo.mfr.equals("0086"))
+		// 4 hour ping for Aeotec
+		sendEvent(name: "checkInterval", value: 4 * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
typically the formula for the checkinterval is `(wakeupinterval)*2 + 2 minutes` which lets us miss one check-in and gives us some wiggle room before marking a device offline.

Reply:
Done.